_specs/ecip-1000.md

```diff
@@ -355,9 +355,9 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
 
 # Appendix: Irregular Process Transitions
 
-As a decentralized community, it is important that for all governance actions, especially in ECIP, the established process is followed. In this section, we document irregular process transitions, and give special permissions to them, to state that they followed the process. Note that this is essentially similar to theDAO hard fork in Ethereum, and should only be added in extreme cases.
+As a decentralized community, it is important that for all governance actions, especially in ECIP, the established process is followed. In this section, we document irregular process transitions, and give special permissions to them, in order to establish them _ex post facto_ as following procedure. Note that this is essentially similar to theDAO hard fork in Ethereum, and should only be added in extreme cases.
 
-* In [PR#199](https://github.com/ethereumclassic/ECIPs/pull/199), Afri Schoedon merged a PR moving ECIP-1061 to "Last Call" status. At the time, Afri dismissed a review of clear objections from ECIP editor Wei Tang, without waiting for consultations of the community, or resolutions with the review submitter. We here declare that this particular action from Afri followed the ECIP process. 
+* [Pull Request #199](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199.json) was [merged](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_event_2837817387.json), [moving it's associated ECIP to `Last Call` status](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199.patch#L21) after [dismissing](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_event_2837816513.json) a [review](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_review_323755348.json) which [provided a reasonable objection](https://github.com/ethereumclassic/ECIPs/blob/gh/.gh-pullrequests/00199_review_323755348.json#L24). This constituted a process deviating from the then-established [procedural specification](TODO).
```

Comment:
> This constituted a process deviating from the then-established [procedural specification](TODO).

Needs to be finished. See https://github.com/ethereumclassic/ECIPs/pull/237#issue-348371179 for WIP reasoning.